rclpy/src/rclpy/_rclpy_logging.cpp

```diff
@@ -63,15 +67,21 @@ rclpy_logging_shutdown()
  *
  * \param[in] name Fully-qualified name of logger.
  * \param[in] level to set
+ * \param[in] detailed_error True for reporting detailed rcutils error.
  * \return None
  */
 void
-rclpy_logging_set_logger_level(const char * name, int level)
+rclpy_logging_set_logger_level(const char * name, int level, bool detailed_error = false)
```

Comment:
I think we can change this `detailed_error` in default even w/o option, since it is better for user application to get the detailed information?

Reply:
If only this function return error log as `rclpy::append_rcutils_error("Failed reason")`, the style of error logs is different from other functions in this file.  
Currently, rclpy::append_rcutils_error("Failed reason") is only for logger service.  
What do you think?